src/presentation/components/CC/Form/CreditCardForm.tsx

```diff
@@ -163,6 +163,10 @@ export const CreditCardForm = () => {
           <div className={styles.creditCardForm__button}>
             <Button type="submit">{t('creditCard.button')}</Button>
           </div>
+          <Typography variant="body1">
+            {t('creditCard.faqText')}
+            <Link href="https://dolphinentertainment.zendesk.com/">{t('creditCard.faq')}</Link>
```

Comment:
we have an env variable with this value. Please use it instead here REACT_APP_ZENDESK_URL